pyannote/audio/models/tdnn.py

```diff
@@ -0,0 +1,186 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
+# The MIT License (MIT)
+
+# Copyright (c) 2019 CNRS
```

Comment:
If this code was taken from elsewhere (MIT License as well?), we need to keep their own copyright. Was there any?

Reply:
The `TDNN` class was taken from [here](https://github.com/jonasvdd/TDNN), which is also MIT.
The rest of the code was written by me.
I am not sure how to add/reference the license of the other project.